src/ts/utils/decoration.ts

```diff
@@ -98,21 +108,17 @@ const createHeightMarkerElement = (id: string) => {
  */
 export const createDecorationsForMatch = (
   match: IMatch,
+  matchColours: IMatchColours = defaultMatchColours,
   isSelected = false,
   addWidgetDecorations = true
 ) => {
   const className = isSelected
-    ? `${DecorationClassMap[DECORATION_MATCH]} ${
-        DecorationClassMap[DECORATION_MATCH_IS_SELECTED]
-      }`
+    ? `${DecorationClassMap[DECORATION_MATCH]} ${DecorationClassMap[DECORATION_MATCH_IS_SELECTED]}`
     : DecorationClassMap[DECORATION_MATCH];
 
-
-  const matchColour = match.markAsCorrect ? "3ff200" :  match.category.colour;
+  const matchColour = getColourForMatch(match, matchColours);
   const opacity = isSelected ? "30" : "07";
-  const style = `background-color: #${
-    matchColour
-  }${opacity}; border-bottom: 2px solid #${matchColour}`;
+  const style = `background-color: ${matchColour}${opacity}; border-bottom: 2px solid ${matchColour}`;
```

Comment:
Did we want to change this to apply specific classes instead of inline styles?

Reply:
As you mention below, the colours can be fed in by the consumer in `createTyperighterPlugin`, so we've an API to modify them that doesn't require classes.